src/Typography/Typography.types.ts

```diff
@@ -0,0 +1,25 @@
+import { ElementType, HTMLAttributes, ReactNode } from 'react';
+export interface TypographyProps
+  extends HTMLAttributes<HTMLHeadingElement>,
+    HTMLAttributes<HTMLParagraphElement> {
+  /**Applies the theme typography styles.*/
+  variant?:
+    | 'hero'
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
+    | 'subtitle'
+    | 'body1'
+    | 'body2'
+    | 'caption'
+    | 'disclaimer';
+  /**The content of the component*/
+  children?: ReactNode;
```

Comment:
Seemed that `children` prop is optional. Is it mentioned what will be the behavior in case we did not pass it ?
and why it is optional  ?

Reply:
For example, It is not mandatory to pass children for <h1></h1>, so why not optional?

Reply:
Do we need to cover this case as per the  given design?.

Reply:
Okay, I changed the Children to a required prop :+1: